modules/core/src/main/java/org/apache/ignite/internal/processors/affinity/GridAffinityProcessor.java

```diff
@@ -1121,6 +1141,22 @@ public CacheAffinityProxy(String cacheName) {
             }
         }
 
+        /** {@inheritDoc} */
+        @Override
+        public List<ClusterNode> mapPartitionToPrimaryAndBackupsList(int part) {
```

Comment:
Any reason to copy-paste implementation from `mapPartitionToPrimaryAndBackups`?

Reply:
no one!